bin/synapse

```diff
@@ -61,6 +61,8 @@ if config.has_key?('service_conf_dir')
   cfiles.each { |x| config['services'][File.basename(x[/(.*)\.(json|yaml)$/, 1])] = parseconfig(x) }
 end
 
+Synapse::StatsD.configure_statsd(config["statsd"] || {})
```

Comment:
should probably allow the `Synapse::Synapse::initialize` method do this configuration. the purpose of the bin is to load the `config` object and pass it to `initialize`, but there are other places where you might want to do that (e.g., tests) where you might forget to call `configure_statsd` beforehand

Reply:
Thanks, that's exactly what I wanted. Putting it here was a result of my limited understanding of the code, and I missed this on my cleanup pass. I'll change that.